index.d.ts

```diff
@@ -25,6 +25,7 @@ declare const tempy: {
 	/**
 	Get a temporary file path you can write to.
 
+	@param filePath - Path of the temporary file.
```

Comment:
This should be an option, not a parameter, and it should have a name that better describes what it does.

The docs should also include the use-case for it, which you can find here: https://github.com/sindresorhus/temp-write/issues/2